doc/source/ray-core/handling-dependencies.rst

```diff
@@ -355,6 +355,8 @@ The ``runtime_env`` is a Python dictionary or a Python class :class:`ray.runtime
 
   Note: If the local directory contains a ``.gitignore`` file, the files and paths specified there are not uploaded to the cluster.  You can disable this by setting the environment variable `RAY_RUNTIME_ENV_IGNORE_GITIGNORE=1` on the machine doing the uploading.
 
+  Note: If the local directory contains symbolic links, the links will be followed and the files they point to will be uploaded to the cluster.
```

Comment:
```suggestion
  Note: If the local directory contains symbolic links, Ray follows the links and the files they point to are uploaded to the cluster.
```

Reply:
Just a nit to remove passive voice. Please consider using Vale to catch these style issues. (go/vale)

Please correct if "Ray" is not the correct subject for the "follows the links" action.